dist/init/mac-launchd/README.md

```diff
@@ -1,12 +1,42 @@
 launchd service for macOS
 =========================
 
-This is a sample file for a *launchd* service on Mac.
-Edit the paths and email in the plist file to match your info.
+This is a working sample file for a *launchd* service on Mac, which should be placed here:
+
+```bash
+/Library/LaunchDaemons/com.caddyserver.web.plist
+```
+
+To create the proper directories as used in the example file:
+
+```bash
+sudo mkdir -p /etc/caddy /etc/ssl/caddy /var/log/caddy /usr/local/bin /var/tmp /srv/www/localhost
+sudo touch /etc/caddy/Caddyfile
+sudo chown root:wheel -R /usr/local/bin/caddy /Library/LaunchDaemons/
+sudo chown _www:_www -R /etc/caddy /etc/ssl/caddy /var/log/caddy
+sudo chmod 0750 /etc/ssl/caddy
+```
+
+Create a simple web page and Caddyfile
+
+```bash
+sudo bash -c 'echo "Hello, World!" > /srv/www/localhost/index.html'
+sudo bash -c 'echo "http://localhost {
+    root /srv/www/localhost
+}" >> /etc/caddy/Caddyfile'
+```
 
 Start and Stop the Caddy launchd service using the following commands:
 
-    $ launchctl load ~/Library/LaunchAgents/com.caddyserver.web.plist
-    $ launchctl unload ~/Library/LaunchAgents/com.caddyserver.web.plist
+```bash
+launchctl load /Library/LaunchDaemons/com.caddyserver.web.plist
+launchctl unload /Library/LaunchDaemons/com.caddyserver.web.plist
```

Comment:
com.caddyserver.caddy?

or com.getcaddy.caddy?

It's reverse dns notation so you shouldn't call it by a name that you don't own.